tests/integration/golang/mlflow/artifact/get_artifact_s3_test.go

```diff
@@ -102,10 +102,7 @@ func (s *GetArtifactS3TestSuite) Test_Ok() {
 
 			// upload artifact subdir object to S3
 			putObjReq = &s3.PutObjectInput{
-				Key: aws.String(fmt.Sprintf(
-					"1/%s/artifacts/artifact.subdir/artifact.file",
-					runID),
-				),
+				Key:    aws.String(fmt.Sprintf("1/%s/artifacts/artifact.subdir/artifact.file", runID)),
```

Comment:
this change is unrelated _and_ makes this less readable (IMHO)

Reply:
yep. ok.

Reply:
done.